docs/usage.md

```diff
@@ -0,0 +1,187 @@
+
+```ts
+import { Realtime } from 'ably/promise';
+import Spaces from '@ably-labs/spaces';
+
+const client = new Realtime(ABLY_API_KEY);
+const spaces = new Spaces(client);
+```
+
+## Create a space
+
+A space is the virtual area of an application you want to monitor, such as a web page, or slideshow. A `space` is uniquely identified by its name. A space is created, or an existing space retrieved from the `spaces` collection by calling the `get()` method. You can only connect to one space in a single operation. The following is an example of creating a space called "demonSlideshow":
+
+```ts
+const space = spaces.get('demoSlideshow');
+```
+
+A set of `spaceOptions` can be passed to space when creating or retrieving it. `spaceOptions` enable additional properties to be set for the space. The following properties can be set:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| offlineTimeout | The time in milliseconds before a member is removed from a space after they have disconnected. The default is 120,000 (2 minutes). | number |
+
+The following is an example of setting `offlineTimeout` to 3 minutes when creating a space:
+
+```ts
+const space = spaces.get('demoSlideshow', { offlineTimeout: 180_000 });
+```
+
+## Subscribe to member updates
+
+Subscribe to `membersUpdate` events in order to display which users are present in a space, such as in an avatar stack. The `membersUpdate` events for a space notify subscribers when clients join and leave it. Use the `space.on()` method to register a listener for `membersUpdate` events.
+
+The following is an example of subscribing to updates for a space:
+
+```ts
+space.on('membersUpdate', (members) => {
+  console.log(members);
+});
+```
+
+The following is an example `membersUpdate` event received by subscribers when a user enters a space:
+
+```json
+[
+  {
+    "clientId": "clemons@slides.com",
+    "isConnected": true,
+    "lastEvent": {
+      "name": "enter",
+      "timestamp": 1677595689759
+    },
+    "location": null,
+    "profileData": {
+      "username": "Claire Lemons",
+      "avatar": "https://slides-internal.com/users/clemons.png"
+    }
+  }
+]
+```
+
+The following are the properties of a `spaceMember` event:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| clientId | The client identifier for the user. | string |
+| isConnected | Whether the user is connected to Ably or not. | boolean |
+| lastEvent | The most recent event emitted by the user and its timestamp. Events will be either `enter` or `leave`. | {name: string, timestamp: number} |
+| location | The current location of the user within the space. | any |
+| profileData | Optional user data that can be attached to a user, such as a username or image to display in an avatar stack. | object |
+
+To stop subscribing to member events, users can call the `space.off()` method.
+
+### Request member updates
```

Comment:
Maybe `Getting current member & self state` ?

Reply:
Would prefer to use something like 'request' for one-off calls, unless you strongly disagree?